zebra-network/src/peer_set/initialize.rs

```diff
@@ -246,10 +246,12 @@ async fn open_listener(config: &Config) -> (TcpListener, SocketAddr) {
     };
     if config.listen_addr.port() == wrong_net.default_port() {
         warn!(
-            "We are configured with port {} for {:?}, but that port is the default port for {:?}",
+            "We are configured with port {} for {:?}, but that port is the default port for {:?}. Default port for {:?} is {}.",
```

Comment:
Minor: grammar suggestion

```suggestion
            "We are configured with port {} for {:?}, but that port is the default port for {:?}. The default port for {:?} is {}.",
```